# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

bool drive
bool active_brake
bool reset
float32 target_speed
